draft-ietf-masque-connect-ip.md

```diff
@@ -1469,6 +1469,13 @@ on-path observer between the IP proxying endpoints. This can potentially expose
 a single end-to-end flow. Because of this, such use of DSCPs in
 privacy-sensitive contexts is NOT RECOMMENDED.
 
+Opportunistic sending of IP packets (see {{link-operation}}) is only allowed in
+versions of HTTP that are greater or equal to 2, because, in HTTP/1.1, a server
+that does not implement this specification could reject the HTTP Upgrade and
+attempt to parse the IP packets as HTTP message content or as a subsequent HTTP
```

Comment:
> as HTTP message content

Is this possible? IIRC connect-ip sends neither Content-Length nor Transfer-encoding, and that makes the HTTP/1.1 server assume that the size of the request body to be zero, assuming that it ignored the upgrade. Therefore, I think all the opportunistic payload will be considered as "subsequent HTTP requests"?

Reply:
Good point, removed message content and kept subsequent requests